lib/options.py

```diff
@@ -193,15 +193,15 @@ def __init__(self):
         parser.add_argument(
                 "--test-timeout",
                 dest="test_timeout",
-                default=110,
+                default=env_int('TEST_TIMEOUT', 110),
```

Comment:
Since those values may be passed implicitly now, I would show them on the terminal before start testing.

Reply:
Sure, added print block, looks like:
```
Running in parallel with 16 workers

Timeout variables:
-------------------
REPLICATION_SYNC_TIMEOUT: 100
TEST_TIMEOUT:             110
NO_OUTPUT_TIMEOUT:        120

Collecting tests in 'app'            (Found 0   tests): application server tests.
```